<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jadwal Booking - Aplikasi Jasa Makeup</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frameSidebar">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frameSidebar {
    background-color: #E91E63;
}

QPushButton {
    background-color: transparent;
    color: white;
    text-align: left;
    padding-left: 20px;
    border: none;
    font-size: 11pt;
}

QPushButton:hover {
    background-color: #C2185B;
}

QLabel {
    color: white;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblLogo">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>üíÑ MAKEUP APP</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDashboard">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üè† Dashboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPelanggan">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üë• Data Pelanggan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLayanan">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>‚ú® Layanan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnJadwal">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üìÖ Jadwal Booking</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTransaksi">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üõí Transaksi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPembayaran">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üí≥ Pembayaran</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetContent" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frameHeader">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame#frameHeader {
    background-color: white;
    border-bottom: 1px solid #ddd;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblPageTitle">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Jadwal Booking</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lblUsername">
            <property name="text">
             <string>üë§ Admin User</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnLogout">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #f44336;
    color: white;
    border: none;
    border-radius: 5px;
    padding: 8px 15px;
}
QPushButton:hover {
    background-color: #da190b;
}</string>
            </property>
            <property name="text">
             <string>Logout</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="pageContent" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget#pageContent {
    background-color: #f5f5f5;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QFrame" name="frameCalendar">
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>350</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame {
    background-color: white;
    border-radius: 8px;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>üìÖ Kalender</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCalendarWidget" name="calendarJadwal"/>
             </item>
             <item>
              <widget class="QPushButton" name="btnBuatJadwal">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: #E91E63;
    color: white;
    padding: 10px 20px;
    border: none;
    border-radius: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #C2185B;
}</string>
               </property>
               <property name="text">
                <string>‚ûï Buat Jadwal Baru</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widgetJadwalRight" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QFrame" name="frameFilter">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Status:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cmbFilterStatus">
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QComboBox {
    border: 1px solid #ddd;
    border-radius: 4px;
    padding: 8px;
}</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>Semua Status</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Pending</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Confirmed</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Completed</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Cancelled</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="tableJadwal">
               <property name="styleSheet">
                <string notr="true">QTableWidget {
    background-color: white;
    border: 1px solid #ddd;
    gridline-color: #f0f0f0;
}
QTableWidget::item:selected {
    background-color: #FCE4EC;
    color: #333;
}
QHeaderView::section {
    background-color: #E91E63;
    color: white;
    padding: 8px;
    border: none;
    font-weight: bold;
}</string>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <column>
                <property name="text">
                 <string>ID</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Tanggal</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Jam Mulai</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Jam Selesai</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Pelanggan</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>MUA</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Status</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Aksi</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupFormBooking">
               <property name="styleSheet">
                <string notr="true">QGroupBox {
    border: 2px solid #E91E63;
    border-radius: 8px;
    margin-top: 10px;
    padding: 15px;
    background-color: white;
    font-weight: bold;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 10px;
    background-color: white;
    color: #E91E63;
}</string>
               </property>
               <property name="title">
                <string>üìù Form Booking</string>
               </property>
               <layout class="QFormLayout" name="formLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Pelanggan:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="cmbPelanggan">
                  <property name="styleSheet">
                   <string notr="true">QComboBox {
    border: 1px solid #ddd;
    border-radius: 4px;
    padding: 8px;
}</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>MUA:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="cmbMUA">
                  <property name="styleSheet">
                   <string notr="true">QComboBox {
    border: 1px solid #ddd;
    border-radius: 4px;
    padding: 8px;
}</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Tanggal:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QDateEdit" name="dateTanggal">
                  <property name="styleSheet">
                   <string notr="true">QDateEdit {
    border: 1px solid #ddd;
    border-radius: 4px;
    padding: 8px;
}</string>
                  </property>
                  <property name="calendarPopup">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Jam Mulai:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QTimeEdit" name="timeJamMulai">
                  <property name="styleSheet">
                   <string notr="true">QTimeEdit {
    border: 1px solid #ddd;
    border-radius: 4px;
    padding: 8px;
}</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Jam Selesai:</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QTimeEdit" name="timeJamSelesai">
                  <property name="styleSheet">
                   <string notr="true">QTimeEdit {
    border: 1px solid #ddd;
    border-radius: 4px;
    padding: 8px;
}</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Status:</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QComboBox" name="cmbStatus">
                  <property name="styleSheet">
                   <string notr="true">QComboBox {
    border: 1px solid #ddd;
    border-radius: 4px;
    padding: 8px;
}</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>Pending</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Confirmed</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Completed</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Cancelled</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QWidget" name="widget" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <spacer name="horizontalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="btnSimpanJadwal">
                     <property name="minimumSize">
                      <size>
                       <width>120</width>
                       <height>35</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QPushButton {
    background-color: #4CAF50;
    color: white;
    border: none;
    border-radius: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #45a049;
}</string>
                     </property>
                     <property name="text">
                      <string>üíæ Simpan</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="btnBatalJadwal">
                     <property name="minimumSize">
                      <size>
                       <width>120</width>
                       <height>35</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QPushButton {
    background-color: #f44336;
    color: white;
    border: none;
    border-radius: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #da190b;
}</string>
                     </property>
                     <property name="text">
                      <string>‚ùå Batal</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
